tests/basic-usage.test.css

```diff
@@ -597,6 +597,15 @@
 .whitespace-nowrap {
   white-space: nowrap;
 }
+.text-wrap {
+  text-wrap: wrap;
+}
+.text-nowrap {
+  text-wrap: nowrap;
+}
+.text-balance {
+  text-wrap: wrap;
```

Comment:
```suggestion
  text-wrap: balance;
```

Reply:
But actually thanks very much 😆 Tests work better when they are asserting the right thing.